src/seedu/addressbook/commands/Command.java

```diff
@@ -43,6 +43,14 @@ public CommandResult execute(){
         throw new UnsupportedOperationException("This method should be implement in child classes");
     }
 
+    /**
+     * Returns boolean value if list has changed
+     * @return
+     */
+    public boolean isMutating() {
```

Comment:
Almost there! I can see that you dont want a default behaviour for this method in the abstract class. How can you achieve this in java?